### PR TITLE
50 improve required field look and feel

### PR DESCRIPTION
**Pull request by daros10**
This PR replaces the previous one.

### CODE REVIEW

File: src/app/modules/activities-management/components/create-activity/create-activity.component.ts [hunk 1]
```diff
@@ -17,7 +16,7 @@ export class CreateActivityComponent {
   constructor(private formBuilder: FormBuilder) {
     this.activityForm = this.formBuilder.group({
       name: ['', Validators.required],
-      description: ['']
+      description: ['', Validators.required],
```
Comment (macrisguncay) [inline src/app/modules/activities-management/components/create-activity/create-activity.component.ts:19]: I don't if the description is required in activity. @enriquezrene can help us.

Comment (enriquezrene) [reply src/app/modules/activities-management/components/create-activity/create-activity.component.ts:19]: it is not.

Comment (enriquezrene) [reply src/app/modules/activities-management/components/create-activity/create-activity.component.ts:19]: It is not, let's remove this validation @daros10

File: src/app/modules/project-management/components/create-project/create-project.component.html [hunk 2]
```diff
@@ -4,26 +4,52 @@ <h1 class="card-title">Project</h1>
...
+      <input
+        class="form-control"
+        id="name"
+        type="text"
+        formControlName="name"
+        [class.is-invalid]="name.invalid && name.touched"
+        required
+      />
+      <p class="text-danger" *ngIf="(name.dirty || name.touched) && name.invalid && name.errors.required">
+        Name Project is required.
```
Comment (enriquezrene) [inline src/app/modules/project-management/components/create-project/create-project.component.html:16]: Project name is required.

File: src/app/modules/project-management/components/create-project/create-project.component.html [hunk 3]
```diff
@@ -4,26 +4,52 @@ <h1 class="card-title">Project</h1>
...
+        class="form-control"
+        rows="3"
+        id="details"
+        type="text"
+        formControlName="details"
+        [class.is-invalid]="details.invalid && details.touched"
+        required
+      ></textarea>
+      <p class="text-danger" *ngIf="(details.dirty || details.touched) && details.invalid && details.errors.required">
+        Details Project is required.
```
Comment (enriquezrene) [inline src/app/modules/project-management/components/create-project/create-project.component.html:32]: Project details are required.

File: src/app/modules/project-management/components/create-project/create-project.component.html [hunk 4]
```diff
@@ -4,26 +4,52 @@ <h1 class="card-title">Project</h1>
...
+        <option
+          *ngFor="let status of projectStatus"
+          [class.is-invalid]="status.invalid && status.touched"
+          [value]="status"
+          >{{ status }}</option
+        >
       </select>
-      <p class="alert alert-danger" *ngIf="(status.dirty || status.touched) && status.invalid && status.errors.required">Status Project is required.</p>
+      <p class="text-danger" *ngIf="(status.dirty || status.touched) && status.invalid && status.errors.required">
+        Status Project is required.
```
Comment (enriquezrene) [inline src/app/modules/project-management/components/create-project/create-project.component.html:47]: Project status is required.

File: src/app/modules/activities-management/components/create-activity/create-activity.component.ts (hunk 1 above)
Comment (daros10) [reply src/app/modules/activities-management/components/create-activity/create-activity.component.ts]: done @enriquezrene

**Pull request merged**